docs/pages/deploy-a-cluster/teleport-enterprise/getting-started.mdx

```diff
@@ -1,465 +1,489 @@
-$ cp ~/downloads/license.pem .
-
-# Download the quickstart file from our GitHub repo
-$ curl -Lso teleport-ent-quickstart.yml https://raw.githubusercontent.com/gravitational/teleport/v(=teleport.version=)/docker/teleport-ent-quickstart.yml
-
-# start teleport quickstart using docker-compose
-$ docker-compose -f teleport-ent-quickstart.yml up
+$ tsh ssh ubuntu@mynode
 ```
 
-- The `docker-compose` quickstart will automatically create a config file for you at `./docker/teleport/config/teleport.yaml`
-- This config is mounted into the container under `/etc/teleport/teleport.yaml`
-- It will also start `teleport` using this config file, with Teleport's data directory set to `./docker/teleport/data` and mounted under `/var/lib/teleport`
-- It will mount your license file (named `license.pem`) from the current directory into the Docker container
-- By default, `docker-compose` will output Teleport's logs to the console for you to observe.
-  - If you would rather run the Teleport container in the background, use `docker-compose -f teleport-ent-quickstart.yml up -d`
-  - You can stop the Teleport container using `docker-compose -f teleport-ent-quickstart.yml down`
+## Next steps
 
-### Quickstart using docker run
+### Deploy on Kubernetes
 
-If you'd prefer to complete these steps manually, here's some sample `docker run` commands:
+This guide shows you how to install Teleport Enterprise on a virtual machine. If
+you are using a Kubernetes-based environment, see our [Getting Started
+Guide](../getting-started/kubernetes-cluster.mdx) for how to deploy Teleport on
+Kubernetes.
 
-```code
-# create local config and data directories for teleport, which will be mounted into the container
-$ mkdir -p ~/teleport/config ~/teleport/data
-
-# download your license file from the Gravitational dashboard and put it in the correct directory
-# the file needs to be named license.pem
-$ cp ~/downloads/license.pem ~/teleport/data/license.pem
-
-# generate a sample teleport config and write it to the local config directory
-# this container will write the config and immediately exit - this is expected
-$ docker run --hostname localhost --rm \
-  --entrypoint=/bin/sh \
-  -v ~/teleport/config:/etc/teleport \
-  (=teleport.latest_ent_docker_image=) -c "teleport configure > /etc/teleport/teleport.yaml"
-
-# change the path to the license file in the sample config
-$ sed -i 's_/path/to/license-if-using-teleport-enterprise.pem_/var/lib/teleport/license.pem_g' ~/teleport/config/teleport.yaml
-
-# start teleport with mounted license, config and data directories, plus all ports
-$ docker run --hostname localhost --name teleport \
-  -v ~/teleport/config:/etc/teleport \
-  -v ~/teleport/data:/var/lib/teleport \
-  -p 3023:3023 -p 3025:3025 -p 3080:3080 \
-  (=teleport.latest_ent_docker_image=)
-```
+### Configure RBAC
 
-### Creating a Teleport user when using Docker quickstart
+The default `access` role we assigned to a user in this guide is probably too
```

Comment:
```suggestion
The preset `access` role we assigned to a user in this guide is probably too
```

Access, Auditor, and Editor are what we call "preset" roles. I think it's important to use that language consistently rather than alternate between "default", "preset", "builtin", etc.